sources/packages/api/src/route-controllers/confirmation-of-enrollment/models/confirmation-of-enrollment.dto.ts

```diff
@@ -29,12 +29,12 @@ export interface ApplicationDetailsForCOEDTO {
   disbursementDate: string;
 }
 
-export interface COEDeniedReasonDto {
+export class COEDeniedReasonAPIOutDTO {
```

Comment:
Not a part of the PR. I assume this class is for loading the dropdown values in a prime vue component. 
We can add a todo to move it to common vuetify dto when we change to vuetify.

Reply:
We already have a generic version called `OptionItemAPIOutDTO`. I had some back and forth about it and end up deciding no change it because maybe this one could be extended with more info.
If I was building it I would use the generic one  `OptionItemAPIOutDTO`, since it is already there, for this case, I just left it as it is right now.